app/views/pages/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="banner">
+  <div class="banner-info">
+    <h1>
+      <%= link_to "create a unique experience", <new_experience_path%></new_experience_path%>
```

Comment:
typo check you have a "<new_experience_path%></" near the new_experience_path

Reply:
ok